docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -8,92 +8,99 @@ ms.author: mikerou
 
 # Migrating WPF Apps to .NET Core
 
-In this guide, we will look at the steps necessary to migrate a Windows Presentation Foundation (WPF) app to .NET Core 3.0. If you don't have a WPF app on hand to port, but would like to try out the process, you can use the **Bean Trader** sample app available on [GitHub][sample-github]. The original app (targeting .NET Framework 4.7.2) is available in the *NetFx\BeanTraderClient* folder. This guide explains the steps necessary to port apps in general, first, and then walks through the specific changes to apply that step to the **Bean Trader** sample.
+In this article, we'll look at the steps necessary to migrate a Windows Presentation Foundation (WPF) app from .NET Framework to .NET Core 3.0. If you don't have a WPF app on hand to port, but would like to try out the process, you can use the **Bean Trader** sample app available on [GitHub](https://github.com/dotnet/windows-desktop/tree/master/Samples/BeanTrader). The original app (targeting .NET Framework 4.7.2) is available in the *NetFx\BeanTraderClient* folder. This guide explains the steps necessary to port apps in general, first, and then walks through the specific changes to apply that step to the **Bean Trader** sample.
 
 [!INCLUDE [desktop guide under construction](../../../includes/desktop-guide-preview-note.md)]
 
 The steps used in this tutorial to migrate to .NET Core are:
 
-01. Prepare by understanding dependencies and updating NuGet dependencies.
+01. Understand and update NuGet dependencies:
+
     01. Upgrade NuGet dependencies to use the `<PackageReference>` format.
     01. Review top-level NuGet dependencies for .NET Core or .NET Standard compatibility.
     01. Upgrade NuGet packages to newer versions, if needed.
-    01. Use the [.NET Portability Analyzer][apiport] to understand .NET dependencies.
-01. Migrate the project file to the new SDK-style format.
-    01. Choose whether to target both .NET Core and .NET Framework or only .NET Core.
+    01. Use the [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md) to understand .NET dependencies.
+
+01. Migrate the project file to the new SDK-style format:
+
+    01. Choose whether to target both .NET Core and .NET Framework, or only .NET Core.
     01. Copy relevant project file properties and items to the new project file.
-01. Fix build issues.
-    01. Add a reference to the [Microsoft.Windows.Compatibility] package.
+
+01. Fix build issues:
+
+    01. Add a reference to the [Microsoft.Windows.Compatibility](https://www.nuget.org/packages/Microsoft.Windows.Compatibility/) package.
     01. Find and fix API-level differences.
     01. Remove app.config sections other than appSettings or connectionStrings.
     01. Regenerate generated code, if needed.
-01. Runtime testing.
+
+01. Runtime testing:
+
     01. Confirm the ported app works as expected.
-    01. Beware of NotSupportedExceptions.
+    01. Beware of <xref:System.NotSupportedException> exceptions.
```

Comment:
I think it will read better like this.

```suggestion
    01. Beware of <xref:System.NotSupportedException>s.
```

Reply:
This reads nicely in English but it doesn't translate well.